HousingSearchListener/V1/UseCase/Interfaces/IAccountAddUseCase.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace HousingSearchListener.V1.UseCase.Interfaces
+{
+    interface IAccountAddUseCase : IMessageProcessing
```

Comment:
To be fair (And I know these interfaces have already been used. But we could just use the basic IMessageProcessing + A factory, and return the right implementation of IMessageProcessing based on a switch statement higher up)